tests/server/TestConfig.js

```diff
@@ -59,15 +59,6 @@ for (const appName of appsList) {
       assert.equal(clientConfig.apiHost, 'https://addons.allizom.org');
       assert.equal(conf.util.getEnv('NODE_ENV'), 'stage');
     });
-
-    it(`should provide a development conf for ${appName}`, () => {
```

Comment:
Since this one is for development I think it's less critical so it can be removed.